openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -15,6 +15,63 @@ function update_len() {
     $('#excerpts-excerpt-len').html(2000 - len).css('color', color);
 }
 
+/**
+ * This is needed because jQuery has no forEach equivalent that works with jQuery elements instead of DOM elements
+ * @param selector - css selector used by jQuery
+ * @returns {*[]} - array of jQuery elements
+ */
+function getJqueryElements(selector){
+    const queryResult = $(selector);
+    return Array.from(queryResult).map((ele,index) => queryResult.eq(index));
+}
+
+export function initLanguageMultiInputAutocomplete() {
+    $(function() {
+        getJqueryElements('.language_multi_input_autocomplete').forEach(jqueryElement => {
+            jqueryElement.setup_multi_input_autocomplete(
+                'input.language-autocomplete',
+                // defined by jsdef
+                // eslint-disable-next-line no-undef
+                render_language_field,
+                {endpoint: '/languages/_autocomplete'},
+                {
+                    max: 6,
+                    // defined by jsdef
+                    // eslint-disable-next-line no-undef
+                    formatItem: render_language_autocomplete_item
+                }
+            );
+        })
+    });
+}
+
+export function initWorksMultiInputAutocomplete() {
+    $(function() {
+        getJqueryElements('.works_multi_input_autocomplete').forEach(jqueryElement => {
```

Comment:
While I imagine we want to eventually wean ourselves off jQuery, we have it now and `$('.works_multi_input_autocomplete').each `would achieve the same with less code.

Reply:
`$('.works_multi_input_autocomplete').each` passes a DOM element to the callback.

To call `setup_multi_input_autocomplete` we need a jQuery element.

I didn't see any simple way to start with a dom element and call `setup_multi_input_autocomplete`. 

The best I could come up with was to use `.eq` inside the each but reading that it wasn't obvious what the code was doing and I didn't want copy/paste an explanation to each place this is used. So I just pulled it out into a function. And that's where `getJqueryElements` came from.

If you have an idea of how to improve this please send the code and we can switch it out.